balancer-js/src/subgraph/graphql/Pools.graphql

```diff
@@ -0,0 +1,216 @@
+query SubgraphPools(
```

Comment:
Do we need to prepend subgraph to each query name? Think it might be cleaner if we remove that and instead, if you want to be explicit about it being a subgraph query, you could just do:
```ts
const subgraphClient = createSubgraphClient(process.env.BALANCER_SUBGRAPH_URL!);
const { pools } = await subgraphClient.Pools({ first: 5 });
```

Reply:
Hey, this is totally fine for me. I'll refactor the queries and remove the `Subgraph` prefix.

What are your thoughts on the types themselves?

```
fragment SubgraphPoolSnapshot on PoolSnapshot {
    id
    pool {
        id
    }
    totalShares
    swapVolume
    swapFees
    timestamp
}
```

It's somehow nice to clearly know that the type is a Subgraph type, and it avoids type collisions with your local types. But, I'm also easy on this and fine with removing the prefix here too.

All types get suffixed with `Fragment` regardless, so that is also an identifying characteristic. IE: the type from the above would be `SubgraphPoolSnapshotFragment`

Reply:
I removed the `Subgraph` prefix from all of the queries:
https://github.com/balancer-labs/balancer-sdk/pull/22/commits/23554fd22358f6e8c58380d06f1482fd2197ef1f
https://github.com/balancer-labs/balancer-sdk/pull/22

Reply:
I agree with using the Subgraph prefix for the types to avoid collisions.